applications/ShapeOptimizationApplication/tests/test_ShapeOptimizationApplication.py

```diff
@@ -67,20 +70,27 @@ def AssembleTestSuites():
 
     # Adding small tests (tests that take < 1s)
     smallSuite = suites['small']
+    smallSuite.addTest(mapper_test('test_execution'))
     smallSuite.addTest(opt_process_vertex_morphing_test('test_execution'))
-    smallSuite.addTest(opt_process_shell_test('test_execution'))
+    if not is_csm_app_missing:
```

Comment:
it would be even nicer to use a 
```python
@KratosUnittest.skipUnless(has_eigensolvers_application,"Missing required application: MeshMovingApplication")
class XYTest():
```
in every individual test. Like this we can see how many tests are skipped, when the tests run.